docs/guides/admin/docs/configuration/elasticsearch.md

```diff
@@ -0,0 +1,23 @@
+Elasticsearch Configuration
+===========================
+
+Elasticsearch powers the external API as well as the administrative user interface of Opencast.
+
+Installing Elasticsearch
+------------------------
+
+The [installation](../installation/index.md) documentation contains instructions for installing Elasticsearch along with your Opencast packaging of choice.
+
+
+Configuring External Nodes
+--------------------------
+
+Opencast's Elasticsearch configuration settings are in the core configuration file, which can usually be found at '/etc/opencast/custom.properties'
```

Comment:
Never describe this location. That's done by docs.opencast.org automatically:
```suggestion
Opencast's Elasticsearch settings can be found in the 'etc/custom.properties' configuration file.
```

Reply:
I didn't know that!  Neat trick!